cmd/panicmonitor/Makefile

```diff
@@ -0,0 +1,6 @@
+panicmonitor:
+	go build -o $@ .
```

Comment:
Moving Makefile to a subdirectory is not a good idea. You can leave Makefile at the root directory, and simply change this to:

```makefile
panicmonitor:
	go build -o $@ github.com/lujjjh/panicmonitor/cmd/panicmonitor
```